TestForSignUp/Program.cs

```diff
@@ -0,0 +1,32 @@
+﻿using OpenQA.Selenium.Chrome;
+using System;
+using System.Threading;
+
+namespace TestForSignUp
+{
+    class Program
```

Comment:
Does visual studio really generates it without access modifier?

Reply:
Yea, VS generate new classes without access modifiers)

Reply:
Denis, don`t forget add access modifiers

Reply:
fixed